e2e-tests/fixtures/Dictionaries.ts

```diff
@@ -72,7 +72,7 @@ export class Dictionaries {
     );
   }
 
-  private async createDictionary(
+  async createDictionary(
```

Comment:
I think the `private` got accidentally removed in this file

Reply:
It gets referenced from outside of the file here: https://github.com/NASA-AMMOS/aerie-ui/blob/bc7e5c2247118f2f9eb17b969bce28583d6bdbce/e2e-tests/tests/parcels.test.ts#L28

Reply:
Ohh. I see. But shouldn't the existing public functions (e.g. `createCommandDictionary`) be used instead?

Reply:
It could if I slightly refactored the API to take the command dictionary name and buffer.

Reply:
Ah I see